src/components/skills-quiz/SearchJobCard.jsx

```diff
@@ -73,10 +73,10 @@ const SearchJobCard = ({ index }) => {
                   {job.job_postings && job.job_postings.length > 0 && (
                     <div>
                       <p className="text-muted m-0 medium-font">
-                        <span style={{ fontWeight: 700 }}>Median Salary:</span> {job.job_postings[0].median_salary}
+                        <span style={{ fontWeight: 500 }}>Median Salary:</span> {job.job_postings[0].median_salary}
```

Comment:
Why we have not moved this styling to `.css` file?
And is there any scenario when there will be no `job_postings` in  `job.job_postings`?

Reply:
Yes, that is handled in upper line` {job.job_postings && job.job_postings.length > 0 && (`